security/certora/specs/Governance.spec

```diff
@@ -368,13 +413,13 @@ rule state_cant_decrease{
   assert assert_uint256(state1) <= assert_uint256(state2);
 }
 
-
+// ND - in the pdf we say three expections but there are four listed 
```

Comment:
The PDF lists four conditions:
https://github.com/Certora/aave-governance-v3/blob/check_proposition_power/security/certora/reports/Formal%20Verification%20Report%20Of%20Aave%20Governance%20V3.pdf